src/content/concepts/why-webpack

```diff
@@ -0,0 +1,55 @@
+---
+title: Why Webpack
+sort: 7
+contributors:
+- debs_obrien
+---
+
+##Why webpack?
```

Comment:
Please leave a space between `#` and the title: `# Why webpack?`

Reply:
done